packages/ui-kit/src/diegesis/flexable-design/BackButton.tsx

```diff
@@ -25,6 +25,13 @@ export type BackBtnConfig = BasicUIConfig & {
     iconColor: string;
   };
 };
+
+export type BackBtnProps = BasicFlexibleProps<BackBtnConfig> & {
+  href?: string;
+  onClick?: (e: any) => void;
+};
+
+//#region data
```

Comment:
Can you plz remove this kinds of comment from the codebase?

//#region (scope)
//#endregion (scope)